rollup-utils/rollup-plugin-sass.js

```diff
@@ -0,0 +1,37 @@
+import sass from 'node-sass';
+import { createFilter } from 'rollup-pluginutils';
+
+export default function plugin (options={}) {
+    let cfg = {
+        // defaults
+        include: [ '**/*.scss' ],
+        exclude: [ 'node_modules/**' ],
+        options: {},
+
+        // overrides
+        ...options
+    };
```

Comment:
break up `cfg` ... unnecessary, given that `options` is being applied with no overrides from function argument